citadel/install_osx_src.md

```diff
@@ -226,7 +226,7 @@ Or in zsh:
 . ~/workspace/install/setup.zsh
 ```
 
-This is the end of the source install instructions; head back to the [Getting started](getting_started.html)
+This is the end of the source install instructions; head back to the [Getting started](../get_started.md)
```

Comment:
So, that's not going to work either, because the page is rendered here https://ignitionrobotics.org/docs/citadel/install_osx_src and not just on GitHub. It needs to link to the actual webpage, not the .md file.

Looking at the Windows source install page, which works https://ignitionrobotics.org/docs/citadel/install_windows_src , it has source here https://github.com/ignitionrobotics/docs/blob/master/citadel/install_windows_src.md , and the way to make it work seems to be `[Getting started](/docs/all/getstarted)`.

Reply:
Ah, okay. thanks for the comment :smile: will do that accordingly.